pkg/dmaasbackup/schedule.go

```diff
@@ -77,12 +83,159 @@ func (d *dmaasBackup) generateBackupName(dbkp v1alpha1.DMaaSBackup) string {
 // updateScheduleInfo checks for relevant velero schedule/backup and
 // update the dbkp with schedule/backup details
 func (d *dmaasBackup) updateScheduleInfo(dbkp *v1alpha1.DMaaSBackup) error {
-	// TODO
+	// check if dbkp is having config for non-scheduled backup
+	if yes := isSchedule(*dbkp); !yes {
+		bkp, err := d.backupClient.Get(dbkp.Name, metav1.GetOptions{})
```

Comment:
can this be avoided by checking that BackupName is already set?

Reply:
yes